### PR TITLE
 vdk-coverity: Adding coverity scan to versatile-data-kit repository.

### PR DESCRIPTION
**Pull request by shanmathik**
Adding Coverity Scan for versatile-data-kit repo as part of SDL procedure.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ stages:
   - release
   - release_image
   - end
+  - coverity-scan
```
Comment (antoniivanov) [inline .gitlab-ci.yml:28]: please put the step before "end" :)

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -51,3 +52,28 @@ notify_failure:
...
+      --form token=$COVERITY_SCAN_TOKEN
+      --form email=$COVERITY_SCAN_EMAIL
+      --form file=@cov-int.tar.gz
+      --form description="Coverity scan for vdk"
+      https://scan.coverity.com/builds?project=$COVERITY_SCAN_PROJECT_NAME
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "schedule" && $IS_COVERITY_SCAN_SCHEDULE == "true"
+  only:
+    refs:
+      - main
```
Comment (antoniivanov) [inline .gitlab-ci.yml:79]: change this to your branch just for testing purposes. you can revert it before merging.

**Comment by antoniivanov** [comment]
You cannot merge from a fork currently. 

Please as you work in VMware. Add yourself to VMware org in Github, then I will add you as contributor to the project and open a PR from a branch instead of a fork. 

Thanks

**Comment by shanmathik** [comment]
Closing this PR.

**Pull request closed without merging**